### PR TITLE
Simplify garage-sign fetching for aktualizr

### PR DESCRIPTION
**Pull request by lbonn**
Two modes:

- `GARAGE_SIGN_AUTOVERSION=0`: use archive and version from aktualizr's recipe
- `GARAGE_SIGN_AUTOVERSION=1`: let aktualizr fetching it automatically

Requires the changes in https://github.com/advancedtelematic/aktualizr/pull/1121

### CODE REVIEW

File: recipes-sota/aktualizr/aktualizr_git.bb [hunk 1]
```diff
@@ -15,14 +13,20 @@ RDEPENDS_${PN}-host-tools = "aktualizr aktualizr-repo aktualizr-cert-provider ${
 PV = "1.0+git${SRCPV}"
 PR = "7"
 
+GARAGE_SIGN_PV = "0.5.0-4-gdd4744f"
```
Comment (patrickvacek) [inline recipes-sota/aktualizr/aktualizr_git.bb]: Why pick a version from five months ago and not the latest? Just for testing?

Comment (lbonn) [reply recipes-sota/aktualizr/aktualizr_git.bb]: sorry, that's what was sitting in one of my build directories... I'll fix

**Comment by lbonn** [comment]
Ok, also includes the necessary aktualizr bump.

**Comment by patrickvacek** [comment]
Passes oe-selftest (finally!) except for raspberry pi (urgh), but that is certainly unrelated. I've also tested manually and it works as expected. Hooray! However, I have one more request: can you add some information about the new variables to the README? I shouldn't need to re-test if you only need to change docs.

**Pull request merged**